platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileWriter.java

```diff
@@ -67,13 +68,25 @@ public static void writeHashInfoFile(final Path directory, final State state) th
         final Path hashInfoFile = directory.resolve(HASH_INFO_FILE_NAME);
 
         try (final BufferedWriter writer = new BufferedWriter(new FileWriter(hashInfoFile.toFile()))) {
-            writer.write(platformInfo);
-            writer.newLine();
             writer.write(hashInfo);
-            writer.flush();
         }
```

Comment:
was the flush deleted intentionally?

Reply:
Yes, it's unnecessary since we are closing the file immediately afterwards (which guarantees that the data makes its way to disk).